### PR TITLE
Alex dev ops

### PR DESCRIPTION
**Pull request by asp78**
This should fix the hanging on the window for creating the tables in the local db.

### CODE REVIEW

**Review by RyanYoung25 — APPROVED** [review]

File: classes/createTable_Classes.py [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+from __future__ import print_function # Python 2/3 compatibility
+import boto3
+import socket
+import time
+import sys
+import botocore
+
+dynamodb = boto3.resource('dynamodb', region_name='us-west-2', endpoint_url='http://localhost:8000')
+
+print("Creating classes table..")
```
Comment (smc395) [inline classes/createTable_Classes.py]: Single quotes

File: classes/createTable_Classes.py [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+# Wait until port is listening
+sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+
+while(sock.connect_ex(('localhost',8000))):
+  print("DB is not up")
```
Comment (smc395) [inline classes/createTable_Classes.py]: single quotes

File: classes/createTable_Classes.py [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+  time.sleep(1)
+
+print("DB is up")
```
Comment (smc395) [inline classes/createTable_Classes.py]: single quotes

File: classes/createTable_Classes.py [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            'AttributeType': 'S'
+        }
+    ],
+    ProvisionedThroughput={
+        'ReadCapacityUnits': 1,
+        'WriteCapacityUnits': 1
+    }
+  )
+  if (usersTable.wait_until_exists()):
+    print("Failed to create classes table", file=sys.stderr)
```
Comment (smc395) [inline classes/createTable_Classes.py]: single quotes

File: createTables.sh [hunk 5]
```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
-echo "Populating DB once it is available..."
+echo "Initializing tables in the db once available..."
```
Comment (smc395) [inline createTables.sh:3]: single quotes?

File: createTables.sh [hunk 6]
```diff
@@ -1,7 +1,7 @@
...
 cd users
-./initTable_Users.sh
+./createTable_Users.py
```
Comment (smc395) [inline createTables.sh:5]: no underscore

File: users/createTable_Users.py [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+from __future__ import print_function # Python 2/3 compatibility
+import boto3
+import socket
+import time
+import sys
+import botocore
+
+dynamodb = boto3.resource('dynamodb', region_name='us-west-2', endpoint_url='http://localhost:8000')
+
+print("Creating users table..")
```
Comment (smc395) [inline users/createTable_Users.py]: single quotes

**Review by smc395 — CHANGES_REQUESTED** [review]

**Review by jonathanmcdaniel — APPROVED** [review]
👍

**Review by RyanYoung25 — APPROVED** [review]

**Review by aaronmsegal — APPROVED** [review]

**Review by smc395 — APPROVED** [review]

**Pull request merged**